common-content/en/module/logic/deduction/index.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Deduction"
+time = 10
```

Comment:
I'm guessing this is at least 60 minutes, if we're expecting people to understand and do their first ever murdle?

```suggestion
time = 60
```

Reply:
I played this with primary age children and I think you'd struggle to get an hour out of it. Half an hour?_